src/test/groovy/com/synacy/gradprogram/spock/exercise/DeliveryServicePec.groovy

```diff
@@ -0,0 +1,49 @@
+package com.synacy.gradprogram.spock.exercise
+
+import spock.lang.Specification
+
+
+class DeliveryServiceSpec extends Specification {
+
+    DeliveryService deliveryService
+    DeliveryRequestRepository deliveryRequestRepository = Mock(DeliveryRequestRepository)
+    DateUtils dateUtils = Mock(DateUtils)
+
+    void setup(){
+        deliveryService = new DeliveryService(deliveryRequestRepository, dateUtils)
+    }
+
+    def "Should Create Delivery Request"() {
+        given:
+        Order order = new Order()
+        dateUtils.getCurrentDate() >> new Date()
+
+        when:
+        DeliveryRequest result = deliveryService.createDelivery(order)
+        then:
+        1 * dateUtils.getCurrentDate() >> new Date()
```

Comment:
No need to assert this since this isn't really what the method is doing and is only part of the "how" the method is doing things.